static/styles/settings.scss

```diff
@@ -1008,6 +1008,10 @@ input[type=checkbox].inline-block {
     min-width: initial !important;
 }
 
+.flexbox div {
+    display: inline-flex;
+}
```

Comment:
Either this belongs in `components.scss` (after a lot of careful thought about potential implications) or it should be scoped to the component it's intended to modify.  Otherwise, it'll potentially break other parts of the site.  

@synicalsyntax can you help @dipu989 get this PR in order?

Reply:
Agreeing with Tim that `.flexbox` isn't being used properly here; is there a reason why you are creating a div element for each item in the flexbox? They can be aligned in better ways.

Reply:
1. This should probably be `flex`, not `inline-flex` (they're block vs inline-block flexbox equivalents)
2. Can you find a more descriptive class name than `flexbox`? It's pretty vague for a specific element, and if we were to turn flexbox into a component class in the future, this would ruin it. A better identifier might be an ID named api_key_buttons.

Reply:
@synicalsyntax 

1) Changing inline-flex to flex is causing elements to take up the entire space present in the area. That's why I have left inline-flex as it is(in the new commit) . But if it's causing problem, then i would change display to flex and at smaller screen size, display will be set to inline flex. I have checked this approach and it's working fine.

2) Changed the class flexbox to id api_key_buttons.

Reply:
@synicalsyntax  @timabbott 
PR is ready for review.